src/map/MarkerIcon.js

```diff
@@ -42,7 +42,7 @@ export const getClusterIcon = (phase, count, active) => {
   if (phase === 0) { // in REGISTERING phase
     className = count ? 'green' : 'yellow'; // green if >0 registered
   } else if (count !== 0) {
-    className = count > 1 ? 'green' : 'yellow'; // if 1 meetup then yellow, if >1 green
+    className = count >= 1 ? 'green' : 'yellow'; // if 0 meetup then yellow, if >=1 green
```

Comment:
it doesn't make sense to check if count !==0 above and then have an else clause here.
With the new logic there is only green and red, no yellow, so you can directly check 
```
} else {
  className = count >=1 ? 'green' : 'red';
}
```

Reply:
Yea, my bad. Should be good now